action.yml

```diff
@@ -1,8 +1,15 @@
 name: 'Find reviewers'
 description: "Find random reviewers"
+inputs:
+  token:
+    description: 'Github token'
+    required: true
+  type:
+    description: 'Event that triggered this action'
+    required: true
 outputs:
   time:
     description: "The time this command ran"
```

Comment:
Understand that this wasn't added by your PR but feel free to remove this `output`